components/globe-component/avatar.js

```diff
@@ -17,6 +12,16 @@ class Avatar {
     this.mesh = new THREE.Object3D()
   }
 
+  /**
+   * Updates the distance from the globe based on the camera z position
+   * @param {int} cameraZ camera Z position
+   */
+  updateDistanceFromGlobe (cameraZ) {
+    [this.mesh.children[0]].forEach(avatar => {
+      console.log(avatar.position.z)
```

Comment:
This only logs the position? then you can probably remove it, right?

Reply:
yes... will remove that